C/Coin-OR/coin-or-common.jl

```diff
@@ -18,7 +18,7 @@ function offset_version(upstream, offset)
 end
 
 # GCC version for building the whole system
-gcc_version = v"6"
+gcc_version = v"12"
```

Comment:
This won't work in any released version of Julia

Reply:
What's the highest I can go?